submission/submission-webapp/src/itest/kotlin/ac/uk/ebi/biostd/itest/assertions/AllInOneSubmissionHelper.kt

```diff
@@ -0,0 +1,49 @@
+package ac.uk.ebi.biostd.itest.assertions
+
+import ac.uk.ebi.biostd.client.integration.web.BioWebClient
+import ac.uk.ebi.biostd.itest.factory.allInOneSubmission
+import ac.uk.ebi.biostd.itest.factory.assertAllInOneSubmissionJson
+import ac.uk.ebi.biostd.itest.factory.assertAllInOneSubmissionTsv
+import ac.uk.ebi.biostd.itest.factory.assertAllInOneSubmissionXml
+import ac.uk.ebi.biostd.persistence.service.SubmissionRepository
+import ebi.ac.uk.asserts.assertThat
+import ebi.ac.uk.model.ExtendedSubmission
+import io.github.glytching.junit.extension.folder.TemporaryFolder
+import org.assertj.core.api.Assertions.assertThat
+import java.nio.file.Paths
+
+internal class AllInOneSubmissionHelper(
+    private val basePath: String,
+    private val submissionRepository: SubmissionRepository
+) {
+    internal fun createAllInOneSubmissionFiles(webClient: BioWebClient, tempFolder: TemporaryFolder) {
```

Comment:
What I do not like about this method is that it only make sense if all submission structure match this one, so basically there is a relation between this and object that is not explicit in code which forces others check this and all submission code in order to make sense of it. In my opinion, methods should do a single thing but do it properly. So if this one is in charge it should create the full submission, it that way it will complete a full consistent task.